[35mDEBUG   [0m asyncio:selector_events.py:64 Using selector: EpollSelector
[35mDEBUG   [0m asyncio:selector_events.py:64 Using selector: EpollSelector
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'mh_9haswoc', '-p', '26781']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - 2025-01-31T00:18:18.089Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -name mh_9haswoc
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -port 26781
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - 2025-01-31T00:18:18.089Z	INFO	mh_9haswoc	status-cli/util.go:55	starting messenger
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'zxxyprnen9', '-p', '38591']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - 2025-01-31T00:18:20.102Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -name zxxyprnen9
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -port 38591
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - 2025-01-31T00:18:20.102Z	INFO	zxxyprnen9	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:20.370] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:20.378] New local node record                    seq=1,738,282,700,377 id=a6c6758ea240182f ip=127.0.0.1 udp=0 tcp=46593
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:20.378] Started P2P networking                   self="enode://8e8cb065ecbf49ca1523f0fb730af7bbaf918c85f36fe8f02c16068a20b8963ac1c5eb969ce879a51c32415dca47676015bd066c5a6e97beb124d234b7f252e3@127.0.0.1:46593?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - ERROR[01-31|00:18:20.379] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:20.379] HTTP server started                      endpoint=127.0.0.1:26781 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:20.399] New local node record                    seq=1,738,282,700,383 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:20.403] New local node record                    seq=1,738,282,700,384 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - 2025-01-31T00:18:20.881Z	INFO	mh_9haswoc	status-cli/util.go:68	account created, key UID: 0x688a0dd34b2d2adee4681cd4f17a9508ef498cee09be130873447ebe29d8d98b
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:20.881] New local node record                    seq=1,738,282,700,385 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - 2025-01-31T00:18:20.914Z	INFO	mh_9haswoc	status-cli/util.go:93	messenger started, public key: 0x04b15e7292bf362404eadcb83437771e3455d295209333647792526b1c0e7b06ea73bbe12bd7b5881a3772169d2f53df8516734b6ea856b5aa70c25fe67d5c7d90
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:22.535] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:22.540] New local node record                    seq=1,738,282,702,539 id=ac252f1b5e3e4151 ip=127.0.0.1 udp=0 tcp=42955
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:22.540] Started P2P networking                   self="enode://da70df162dc3cf2e4ffff0a7c600087c8b7139b66630b28380a3a926a5179fa9ef0a9652ef0f65df308cdb7585020a675501734397fcd292153393080558594f@127.0.0.1:42955?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - ERROR[01-31|00:18:22.543] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:22.544] HTTP server started                      endpoint=127.0.0.1:38591 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:22.574] New local node record                    seq=1,738,282,702,548 id=2eeeb58732c63c5a ip=10.1.0.43 udp=0 tcp=46063
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:22.576] New local node record                    seq=1,738,282,702,549 id=2eeeb58732c63c5a ip=10.1.0.43 udp=34208 tcp=46063
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:22.579] New local node record                    seq=1,738,282,702,550 id=2eeeb58732c63c5a ip=10.1.0.43 udp=34208 tcp=46063
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - 2025-01-31T00:18:23.061Z	INFO	zxxyprnen9	status-cli/util.go:68	account created, key UID: 0x3066219f9534a6ab9f3efeb1e84f246d2be5e61b0ee6ca6e72aa76c0b43a7820
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:23.062] New local node record                    seq=1,738,282,702,551 id=2eeeb58732c63c5a ip=10.1.0.43 udp=34208 tcp=46063
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - 2025-01-31T00:18:23.092Z	INFO	zxxyprnen9	status-cli/util.go:93	messenger started, public key: 0x04ae6e27e211cc7965549aeae62e4f1807c44f598e664ab3fd24b56d8cf97021545b5cbca43b35e3b5fc13f3a1356b71d5d55c7a4e317b7ebb35448e3e23a45013
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:25.396] New local node record                    seq=1,738,282,700,386 id=1acac43f00390ab2 ip=20.161.77.227 udp=40466 tcp=10242
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - 2025-01-31T00:18:25.915Z	INFO	mh_9haswoc	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:27.570] New local node record                    seq=1,738,282,702,552 id=2eeeb58732c63c5a ip=20.161.77.227 udp=34208 tcp=10240
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - 2025-01-31T00:18:28.092Z	INFO	zxxyprnen9	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'tx7im-7rv-', '-p', '15036']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - 2025-01-31T00:18:28.202Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -name tx7im-7rv-
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -port 15036
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - 2025-01-31T00:18:28.202Z	INFO	tx7im-7rv-	status-cli/util.go:55	starting messenger
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'nmsxhx_g_e', '-p', '56691']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - 2025-01-31T00:18:30.216Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -name nmsxhx_g_e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -port 56691
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - 2025-01-31T00:18:30.216Z	INFO	nmsxhx_g_e	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:30.559] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:30.566] New local node record                    seq=1,738,282,710,565 id=9436ae22bb7d2a43 ip=127.0.0.1 udp=0 tcp=46779
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:30.566] Started P2P networking                   self="enode://bddaa7807e1fcac3eb1704964eed1b182f63c9823239c49e1b13af9ca69d6223fba0da0c76d10afd7f3414a98278e7ce37ca48fcede9ab8be8bdb28f30093874@127.0.0.1:46779?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - ERROR[01-31|00:18:30.567] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:30.568] HTTP server started                      endpoint=127.0.0.1:15036 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:30.599] New local node record                    seq=1,738,282,710,577 id=0344c80f06525274 ip=10.1.0.43 udp=0 tcp=43083
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:30.601] New local node record                    seq=1,738,282,710,578 id=0344c80f06525274 ip=10.1.0.43 udp=49884 tcp=43083
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:30.602] New local node record                    seq=1,738,282,710,579 id=0344c80f06525274 ip=10.1.0.43 udp=49884 tcp=43083
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - 2025-01-31T00:18:31.130Z	INFO	tx7im-7rv-	status-cli/util.go:68	account created, key UID: 0x17ce556dcb65cd10bfa9a35700c75970da87dd27b70cfc8293fdacde6ce810b0
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:31.131] New local node record                    seq=1,738,282,710,580 id=0344c80f06525274 ip=10.1.0.43 udp=49884 tcp=43083
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - 2025-01-31T00:18:31.172Z	INFO	tx7im-7rv-	status-cli/util.go:93	messenger started, public key: 0x04e23b105a3aa128d81c1186a610766483f5fccc139fa585d51ca50d39417f47542b35866d9ffeb5f9bf3411628cb9fa2e5bf8ddf35117f030c06358afc467d486
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:32.797] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - ERROR[01-31|00:18:32.804] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:32.804] New local node record                    seq=1,738,282,712,803 id=38201bd4d5253a1c ip=127.0.0.1 udp=0 tcp=42207
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:32.804] Started P2P networking                   self="enode://2b0ddd880cdd414215848ba518a059532ccb7b7cd4bb0740bd16b257b42020fb2e649d0d84f90a174a4b5093ec561b353514cfc0040c0695e7761714426fe8db@127.0.0.1:42207?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:32.805] HTTP server started                      endpoint=127.0.0.1:56691 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:32.827] New local node record                    seq=1,738,282,712,810 id=dd7882b56499fe64 ip=10.1.0.43 udp=0 tcp=37595
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:32.828] New local node record                    seq=1,738,282,712,811 id=dd7882b56499fe64 ip=10.1.0.43 udp=58845 tcp=37595
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:32.830] New local node record                    seq=1,738,282,712,812 id=dd7882b56499fe64 ip=10.1.0.43 udp=58845 tcp=37595
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - 2025-01-31T00:18:33.275Z	INFO	nmsxhx_g_e	status-cli/util.go:68	account created, key UID: 0x16d54fdbee376cf4259c8f0c342f84ff9dcbc6dd774ca55f95d6e78a6e29ab51
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:33.275] New local node record                    seq=1,738,282,712,813 id=dd7882b56499fe64 ip=10.1.0.43 udp=58845 tcp=37595
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - 2025-01-31T00:18:33.304Z	INFO	nmsxhx_g_e	status-cli/util.go:93	messenger started, public key: 0x04dbe1f8f18bc558f5959f7b9c392c138e1b4784808fd69fb7e1e4d0c53fedb63382d3183060a16cc733785136ee67aeb575d872e0cff3f70dc5cba29cf22bcffa
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:35.396] New local node record                    seq=1,738,282,700,387 id=1acac43f00390ab2 ip=20.161.77.227 udp=40466 tcp=10240
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:18:35.596] New local node record                    seq=1,738,282,710,581 id=0344c80f06525274 ip=20.161.77.227 udp=49884 tcp=10240
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - 2025-01-31T00:18:36.172Z	INFO	tx7im-7rv-	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:37.318] New local node record                    seq=1,738,282,700,388 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:37.387] New local node record                    seq=1,738,282,700,389 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:18:37.389] New local node record                    seq=1,738,282,700,390 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - INFO [01-31|00:18:37.824] New local node record                    seq=1,738,282,712,814 id=dd7882b56499fe64 ip=20.161.77.227 udp=58845 tcp=10243
[35mDEBUG   [0m src.node.status_node:status_node.py:65 NMSXHX_G_E - 2025-01-31T00:18:38.305Z	INFO	nmsxhx_g_e	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'ws9jqed1ud', '-p', '27067']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - 2025-01-31T00:18:38.417Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -name ws9jqed1ud
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -port 27067
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - 2025-01-31T00:18:38.417Z	INFO	ws9jqed1ud	status-cli/util.go:55	starting messenger
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'p7uc6n26a8', '-p', '12194']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - 2025-01-31T00:18:40.429Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -name p7uc6n26a8
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -port 12194
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - 2025-01-31T00:18:40.429Z	INFO	p7uc6n26a8	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:40.866] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:40.872] New local node record                    seq=1,738,282,720,871 id=2fc30a5e9ee69b0a ip=127.0.0.1 udp=0 tcp=41847
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:40.872] Started P2P networking                   self="enode://5ff0f3fe95bcd774052efcf3e92a274a3c9781874202a8d775c00638c8afc02a1e3617c2f1cafb4382267345a6f21896ff8e39df1908fa7a6f9fad20b0135af7@127.0.0.1:41847?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - ERROR[01-31|00:18:40.873] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:40.874] HTTP server started                      endpoint=127.0.0.1:27067 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:40.904] New local node record                    seq=1,738,282,720,883 id=681eda98b5e94467 ip=10.1.0.43 udp=0 tcp=35857
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:40.905] New local node record                    seq=1,738,282,720,884 id=681eda98b5e94467 ip=10.1.0.43 udp=38065 tcp=35857
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:40.908] New local node record                    seq=1,738,282,720,885 id=681eda98b5e94467 ip=10.1.0.43 udp=38065 tcp=35857
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - 2025-01-31T00:18:41.395Z	INFO	ws9jqed1ud	status-cli/util.go:68	account created, key UID: 0x274de060750243244a9eea0b5bd93311c50e415ec2c6809f11823f65b402493b
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:41.395] New local node record                    seq=1,738,282,720,886 id=681eda98b5e94467 ip=10.1.0.43 udp=38065 tcp=35857
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - 2025-01-31T00:18:41.433Z	INFO	ws9jqed1ud	status-cli/util.go:93	messenger started, public key: 0x044e7e92c9ecd68d82e88f90e35565379463fca122a77539d7ba1dd82e14a378acdcb96d1d2baecfb9a705e22dfadf2456c0ecd2f973ba25f070a4e3e9f12c9b12
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:42.958] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:42.965] New local node record                    seq=1,738,282,722,963 id=339c3b399e2a2b33 ip=127.0.0.1 udp=0 tcp=34929
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:42.965] Started P2P networking                   self="enode://b22d179d448a8be9953b9d6b4f8c196996d3c89b469ee8df7d5fa3345001198aafc28d0753333a2a2b620ef3729c05355668c1d5a500477cbdb3048c47e086aa@127.0.0.1:34929?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - ERROR[01-31|00:18:42.966] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:42.966] HTTP server started                      endpoint=127.0.0.1:12194 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:42.987] New local node record                    seq=1,738,282,722,970 id=9aaa2715180f8491 ip=10.1.0.43 udp=0 tcp=33519
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:42.989] New local node record                    seq=1,738,282,722,971 id=9aaa2715180f8491 ip=10.1.0.43 udp=44646 tcp=33519
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:42.991] New local node record                    seq=1,738,282,722,972 id=9aaa2715180f8491 ip=10.1.0.43 udp=44646 tcp=33519
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - 2025-01-31T00:18:43.447Z	INFO	p7uc6n26a8	status-cli/util.go:68	account created, key UID: 0xb59f34e031b664a82773218dbc7d92dbb9c2cbc6d85a9248d94d0201cd5ff456
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:43.447] New local node record                    seq=1,738,282,722,973 id=9aaa2715180f8491 ip=10.1.0.43 udp=44646 tcp=33519
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - 2025-01-31T00:18:43.479Z	INFO	p7uc6n26a8	status-cli/util.go:93	messenger started, public key: 0x042d424219a3439f5c7bc06e3a43e3e934fd45ce327ee9c064fa24be4e21a25b4dfdf28a95b1f4124e8c615490a5fbb36235cc60caf1f883fe27e15cdc953c4efc
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - INFO [01-31|00:18:45.900] New local node record                    seq=1,738,282,720,887 id=681eda98b5e94467 ip=20.161.77.227 udp=38065 tcp=10244
[35mDEBUG   [0m src.node.status_node:status_node.py:65 WS9JQED1UD - 2025-01-31T00:18:46.434Z	INFO	ws9jqed1ud	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:18:47.570] New local node record                    seq=1,738,282,702,553 id=2eeeb58732c63c5a ip=20.161.77.227 udp=34208 tcp=10240
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - INFO [01-31|00:18:47.984] New local node record                    seq=1,738,282,722,974 id=9aaa2715180f8491 ip=20.161.77.227 udp=44646 tcp=10245
[35mDEBUG   [0m src.node.status_node:status_node.py:65 P7UC6N26A8 - 2025-01-31T00:18:48.479Z	INFO	p7uc6n26a8	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'oty10prgd-', '-p', '8548']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - 2025-01-31T00:18:48.528Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -name oty10prgd-
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -port 8548
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - 2025-01-31T00:18:48.528Z	INFO	oty10prgd-	status-cli/util.go:55	starting messenger
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'enlr42o_vg', '-p', '41632']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - 2025-01-31T00:18:50.534Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -name enlr42o_vg
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -port 41632
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - 2025-01-31T00:18:50.534Z	INFO	enlr42o_vg	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:50.990] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:51.000] New local node record                    seq=1,738,282,730,999 id=9bbbdb2dd94877b3 ip=127.0.0.1 udp=0 tcp=36645
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:51.000] Started P2P networking                   self="enode://df83bd88486e64ad3ba526cb32f06b7e720ab2dd6008c4c54127fd462be0f2286d5201a0c805be6ddccf3efdb341a0297cc073b8c9234bc613e6c6c48825afd6@127.0.0.1:36645?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - ERROR[01-31|00:18:51.001] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:51.002] HTTP server started                      endpoint=127.0.0.1:8548 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:51.029] New local node record                    seq=1,738,282,731,005 id=db9d04793eb00eaf ip=10.1.0.43 udp=0 tcp=36239
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:51.030] New local node record                    seq=1,738,282,731,006 id=db9d04793eb00eaf ip=10.1.0.43 udp=59435 tcp=36239
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:51.032] New local node record                    seq=1,738,282,731,007 id=db9d04793eb00eaf ip=10.1.0.43 udp=59435 tcp=36239
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - 2025-01-31T00:18:51.559Z	INFO	oty10prgd-	status-cli/util.go:68	account created, key UID: 0x26c486c5703d12b1f8aa3993ee70e4d6152072bfd048dcd4a647d0da940fcc6e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:51.560] New local node record                    seq=1,738,282,731,008 id=db9d04793eb00eaf ip=10.1.0.43 udp=59435 tcp=36239
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - 2025-01-31T00:18:51.589Z	INFO	oty10prgd-	status-cli/util.go:93	messenger started, public key: 0x0483a9260626fcc73b84c6e124e415861ca21d827df3bbbda7ccefff3f664271b997e01169d829ff7101576fa717d4d8bbe0887318fbd9ddeee5a8c0f308b26c48
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - INFO [01-31|00:18:53.124] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.21.10
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - INFO [01-31|00:18:53.131] New local node record                    seq=1,738,282,733,130 id=0164cc2bd8a48370 ip=127.0.0.1 udp=0 tcp=42451
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - INFO [01-31|00:18:53.131] Started P2P networking                   self="enode://6f583c5c049d05f7e67dd954f4f6daac1705a14b04cfcda1f49cf9f2012b2271cfa8a7ed60ab2d9117858e2d9e3b099999a6668bfd3c3bf61fe8ab61e43ccb1e@127.0.0.1:42451?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - ERROR[01-31|00:18:53.134] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ENLR42O_VG - 2025/01/31 00:18:55 listen tcp 127.0.0.1:41632: bind: address already in use
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - INFO [01-31|00:18:56.025] New local node record                    seq=1,738,282,731,009 id=db9d04793eb00eaf ip=20.161.77.227 udp=59435 tcp=10246
[35mDEBUG   [0m src.node.status_node:status_node.py:65 OTY10PRGD- - 2025-01-31T00:18:56.589Z	INFO	oty10prgd-	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:19:03.789] New local node record                    seq=1,738,282,702,554 id=2eeeb58732c63c5a ip=10.1.0.43 udp=34208 tcp=46063
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:19:03.873] New local node record                    seq=1,738,282,702,555 id=2eeeb58732c63c5a ip=10.1.0.43 udp=34208 tcp=46063
[35mDEBUG   [0m src.node.status_node:status_node.py:65 ZXXYPRNEN9 - INFO [01-31|00:19:03.875] New local node record                    seq=1,738,282,702,556 id=2eeeb58732c63c5a ip=10.1.0.43 udp=34208 tcp=46063
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:19:10.401] New local node record                    seq=1,738,282,700,391 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 MH_9HASWOC - INFO [01-31|00:19:10.403] New local node record                    seq=1,738,282,700,392 id=1acac43f00390ab2 ip=10.1.0.43 udp=40466 tcp=41739
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:19:14.436] New local node record                    seq=1,738,282,710,582 id=0344c80f06525274 ip=10.1.0.43 udp=49884 tcp=43083
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:19:14.670] New local node record                    seq=1,738,282,710,583 id=0344c80f06525274 ip=10.1.0.43 udp=49884 tcp=43083
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:19:14.672] New local node record                    seq=1,738,282,710,584 id=0344c80f06525274 ip=10.1.0.43 udp=49884 tcp=43083
[35mDEBUG   [0m src.node.status_node:status_node.py:65 TX7IM-7RV- - INFO [01-31|00:19:15.596] New local node record                    seq=1,738,282,710,585 id=0344c80f06525274 ip=20.161.77.227 udp=49884 tcp=10240
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file tx7im-7rv-.log to test_results/testcontacrequest_test_contact_request_baseline_tx7im-7rv-.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file oty10prgd-.log to test_results/testcontacrequest_test_contact_request_baseline_oty10prgd-.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file zxxyprnen9.log to test_results/testcontacrequest_test_contact_request_baseline_zxxyprnen9.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file enlr42o_vg.log to test_results/testcontacrequest_test_contact_request_baseline_enlr42o_vg.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file mh_9haswoc.log to test_results/testcontacrequest_test_contact_request_baseline_mh_9haswoc.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file p7uc6n26a8.log to test_results/testcontacrequest_test_contact_request_baseline_p7uc6n26a8.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file nmsxhx_g_e.log to test_results/testcontacrequest_test_contact_request_baseline_nmsxhx_g_e.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file ws9jqed1ud.log to test_results/testcontacrequest_test_contact_request_baseline_ws9jqed1ud.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: mh_9haswoc
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: zxxyprnen9
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: tx7im-7rv-
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: nmsxhx_g_e
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: ws9jqed1ud
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: p7uc6n26a8
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: oty10prgd-
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: enlr42o_vg
[35mDEBUG   [0m asyncio:selector_events.py:64 Using selector: EpollSelector